massa-protocol-worker/src/connectivity.rs

```diff
@@ -259,59 +261,82 @@ pub(crate) fn start_connectivity_thread(
                         }).count());
                         let mut addresses_to_connect: Vec<SocketAddr> = Vec::new();
                         {
-                            let peer_db_read = peer_db.read();
-                            for (_, peer_id) in &peer_db_read.index_by_newest {
-                                if peers_connected.contains_key(peer_id) {
-                                    continue;
-                                }
-                                if let Some(peer_info) = peer_db_read.peers.get(peer_id).and_then(|peer| {
-                                    if peer.state == PeerState::Trusted {
-                                        Some(peer.clone())
-                                    } else {
-                                        None
-                                    }
-                                }) {
-                                    if peer_info.last_announce.listeners.is_empty() {
+                                let peer_db_read = peer_db.read();
```

Comment:
There's an indentation here that makes it a big diff where there isn't need for one
Maybe try formatting the code ?